README.md

```diff
@@ -30,8 +30,9 @@ Define a `deployment` and an `api` resource. A `deployment` specifies a set of A
 
 - kind: api
   name: generator
-  model: s3://cortex-examples/text-generator/gpt-2/124M
```

Comment:
We should discuss, but we might want to leave the README as is until we release v0.10